### DISCUSSION TITLE
cairo and sysinfo in bar

### DISCUSSION THREAD

**Discussion by leoVici9** [JakeStanger/ironbar#646] [Show and tell]
Just to share my efforts of the last week to make ironbar and hyprland a workable environment for me.  To many things to mention but:
ironbar:  with a bar on the background/wallpaper to display clock and sys_info
inLyne: to display markdown help 
onagre: as launcher
hyprspace: plugin to show workspace overview
SwayOSD for onscreen display for volume
clipse for clipboard.
waypaper ans swww for wallpaper
and lost more ........

![screenshot_22062024_200717-s](https://github.com/JakeStanger/ironbar/assets/160249523/724da144-6723-4477-9fec-c9edd3398b62)
![screenshot_22062024_201216-s](https://github.com/JakeStanger/ironbar/assets/160249523/3a511452-7525-4ac6-a2e1-86608824d2dd)

let me know what you think :-)

**Comment by quynguyencanalcircle** [comment]
Look beautiful!

**Comment by quynguyencanalcircle** [comment]
> Just to share my efforts of the last week to make ironbar and hyprland a workable environment for me. To many things to mention but: ironbar: with a bar on the background/wallpaper to display clock and sys_info inLyne: to display markdown help onagre: as launcher hyprspace: plugin to show workspace overview SwayOSD for onscreen display for volume clipse for clipboard. waypaper ans swww for wallpaper and lost more ........
> 
> ![screenshot_22062024_200717-s](https://private-user-images.githubusercontent.com/160249523/341989981-724da144-6723-4477-9fec-c9edd3398b62.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTkyMTE1MzMsIm5iZiI6MTcxOTIxMTIzMywicGF0aCI6Ii8xNjAyNDk1MjMvMzQxOTg5OTgxLTcyNGRhMTQ0LTY3MjMtNDQ3Ny05ZmVjLWM5ZWRkMzM5OGI2Mi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNjI0JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDYyNFQwNjQwMzNaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT01MmNlNjkzODU2NWJiMzM1NmRjOGJkNWZmZDU0ODljODJlZTRkMWM2YTkwNGFlMTM5NzRmYzZkYzVkOThhYzgzJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.3ebhCLsiOL85nv2s2s2cyN0KRZpzb4-3uwwjzzbmYFg) ![screenshot_22062024_201216-s](https://private-user-images.githubusercontent.com/160249523/341989983-3a511452-7525-4ac6-a2e1-86608824d2dd.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTkyMTE1MzMsIm5iZiI6MTcxOTIxMTIzMywicGF0aCI6Ii8xNjAyNDk1MjMvMzQxOTg5OTgzLTNhNTExNDUyLTc1MjUtNGFjNi1hMmUxLTg2NjA4ODI0ZDJkZC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNjI0JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDYyNFQwNjQwMzNaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0zN2VhOGU5ZDBkZTcxN2QwNjBlZmYwYzU1Y2E0NzU4OTdmZWI4N2U3YjQwZDg2OWZlNTg5MDQwZjQwZmYzYTQyJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.F5OonqU6xEAZgQk4YhgXjeX2tD_v8vCg9opkW2Q9P7k)
> 
> let me know what you think :-)

Can you share your config?

**Reply by leoVici9** [reply]
see attached 
[config.zip](https://github.com/user-attachments/files/15959805/config.zip)

**Reply by quynguyencanalcircle** [reply]
> see attached [config.zip](https://github.com/user-attachments/files/15959805/config.zip)


I'm trying your configs. Thanks!

**Comment by JakeStanger** [comment]
This is really cool, thanks for sharing!
